dxr/plugins/descriptor/__init__.py

```diff
@@ -107,7 +107,8 @@ def generic_describe(self):
         [delimiter] text, and return the first text we find. Unless it's a
         readme, then return the first line."""
 
-        filename = basename(self.path)
+        utf8_path = self.path.encode('utf-8')
```

Comment:
This undoes the decode in build.py, we shouldn't be reassigning there (note to self)